menagerie/concept-shapes/scripts/mint_trinity.py

```
@@ -0,0 +1,1006 @@
+    "key_id": "UNSIGNED_DEV_ONLY",
+    "sig_b64": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+}
+
+
+def compute_cid(memento):
+    with tempfile.NamedTemporaryFile("w", suffix=".json", delete=False) as f:
+        json.dump(memento, f, ensure_ascii=True)
+        f.write("\n")
+        tmp = f.name
+    try:
+        result = subprocess.run([str(BINARY), tmp], capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)